sharepoint-adfs/dsc/ConfigureDCVM.ps1

```diff
@@ -0,0 +1,358 @@
+﻿configuration ConfigureDCVM 
+{ 
+    param 
+    ( 
+        [Parameter(Mandatory)]
+        [String]$DomainFQDN,
+
+        [Parameter(Mandatory)]
+        [System.Management.Automation.PSCredential]$Admincreds,
+
+        [Parameter(Mandatory)]
+        [System.Management.Automation.PSCredential]$AdfsSvcCreds,
+
+        [Parameter(Mandatory)]
+        [String]$PrivateIP,
+
+        [Int] $RetryCount=20,
+        [Int] $RetryIntervalSec=30,
+        [String] $SPTrustedSitesName = "SPSites",
+        [String] $ADFSSiteName = "ADFS"
+    ) 
+    
+    Import-DscResource -ModuleName xActiveDirectory, xDisk, xNetworking, cDisk, xPSDesiredStateConfiguration, xAdcsDeployment, xCertificate, xPendingReboot, cADFS, xDnsServer
+    [String] $DomainNetbiosName = (Get-NetBIOSName -DomainFQDN $DomainFQDN)
+    [System.Management.Automation.PSCredential] $DomainCredsNetbios = New-Object System.Management.Automation.PSCredential ("${DomainNetbiosName}\$($Admincreds.UserName)", $Admincreds.Password)
+    [System.Management.Automation.PSCredential] $AdfsSvcCredsQualified = New-Object System.Management.Automation.PSCredential ("${DomainNetbiosName}\$($AdfsSvcCreds.UserName)", $AdfsSvcCreds.Password)
+    $Interface = Get-NetAdapter| Where-Object Name -Like "Ethernet*"| Select-Object -First 1
+    $InterfaceAlias = $($Interface.Name)
+    $ComputerName = Get-Content env:computername
+
+    Node localhost
```

Comment:
Indentation is off in spots of this file (might be a tabs vs. space problem) - can you load into an editor (e.g. ISE) and fix the indenting?

Reply:
I converted all tabs to spaces and noticed that some lines had an extra space before the line return (which I removed), I hope this is what you meant?

Reply:
yep